apps/src/templates/certificates/certificate_batch.module.scss

```diff
@@ -267,3 +267,37 @@ a.linkButton {
 .catalogButton {
   margin: 20px 0 40px 0 !important;
 }
+
+.studentButtonsContainer {
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+}
+
+.courseContainer {
+  margin-bottom: 35px;
+}
+
+.container {
+  width: 100%;
+  max-width: $width-md;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.divider {
+  border-color: $lightest_gray;
+  border-width: 1px 0 0 0;
+  border-style: solid;
+  margin: 20px 0 20px 0;
```

Comment:
Nit: you can specify top/bottom, left/right if they're the same, so this line can be `margin: 20px 0`